meta-resin-common/recipes-support/resin-supervisor/files/update-resin-supervisor

```diff
@@ -68,5 +68,5 @@ systemctl start resin-supervisor
 
 # Mark supervisor as working. This version will run when the device reboots.
 echo "Mark pulled tag as latest..."
-rce tag -f "$image_name:$tag" "$image_name:latest"
+docker tag -f "$image_name:$tag" "$image_name:latest"
 sed --in-place "s|SUPERVISOR_IMAGE=.*|SUPERVISOR_IMAGE=$image_name|" /etc/supervisor.conf
```

Comment:
Please test a supervisor update just to be sure. @Page- can page to activate the database for a testing device.